tests/converter/test_rectangle.py

```diff
@@ -1,31 +1,29 @@
 from converter.rectangle import convert
-from sketchformat.style import *
 from .base import FIG_BASE
 
 
-def test_straight_corners():
-    rect = convert({**FIG_BASE})
-    for p in rect.points:
-        assert p.cornerRadius == 0
+class TestConvert:
```

Comment:
I'd name this something like TestCorners, but that's a nitpick if I ever heard one.